README.md

```diff
@@ -129,3 +129,38 @@ done that.
 You may have the Perl PAR Packager installed. Use long name `panicparse` then;
 
     go get github.com/maruel/panicparse
+    
+### You can also use it as a library to log your errors like this
```

Comment:
This is a good idea but this belongs to godoc, so it should be in a Example function, not in the README. See section "Examples" in https://golang.org/pkg/testing/.

Thanks